content/blog/refactor-step-tdd-three-line-discipline-ruby/index.md

```
@@ -0,0 +1,241 @@
+    @line_totals << price * quantity
+  end
+
+  def total
+    @line_totals.sum
+  end
+end
+```
+
+Run `bin/rake test:critical`. Four green tests. Commit:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)